vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -253,17 +253,24 @@ function Install-ContainerD {
     Write-Log "Getting containerD binaries from $global:containerdPackageUrl"
 
     $installDir = "c:\program files\containerd"
-    $zipPath = [IO.Path]::Combine($installDir, "containerd.zip")
-
     Write-Log "Installing containerd to $installDir"
     New-Item -ItemType Directory $installDir -Force | Out-Null
-    DownloadFileWithRetry -URL $global:containerdPackageUrl -Dest $zipPath
-    Expand-Archive -Path $zipPath -DestinationPath $installDir
-    Remove-Item -Path $zipPath | Out-null
 
-    $newPath = [Environment]::GetEnvironmentVariable("Path", [EnvironmentVariableTarget]::Machine) + ";$installDir"
-    [Environment]::SetEnvironmentVariable("Path", $newPath, [EnvironmentVariableTarget]::Machine)
-    $env:Path += ";$installDir"
+    if ($global:containerdPackageUrl.endswith(".zip")) {
```

Comment:
If we can use $containerdFilename=[IO.Path]::GetFileName($global:containerdPackageUrl), it should avoid below duplicate code lines.

Reply:
Thanks, this will remove lines of code.